docs/downloads/labels.cm

```diff
@@ -0,0 +1,70 @@
+# -*- mode: yaml -*-
+manifest:
+      version: 1.0
+
+automations:
+  # Apply an update labels that indicate how long it takes to review the PR
+  # This uses the `calc` custom expression found at the bottom of this file.
+  estimated_time_to_review:
+    # Triggered for all new PRs and PR changes
+    if:
+      - true
+    # Apply a color-coded label that indicates how long it would take someone to review the PR
+    run:
+      - action: add-label@v1
+        args:
+          label: "{{ calc.etr }} min review"
+          color: {{ 'E94637' if (calc.etr >= 20) else ('FBBD10' if (calc.etr >= 5) else '36A853') }}
+
+  # Label PRs that are missing required tests.
+  label_missing_tests:
+    # Triggered for changes to the lib and include directories 
+    # when there is no corresponding change in the tests directory
+    # Update "lib|include" and "tests" to match your repository's file structure
+    if:
+      - {{ files | match(regex=r/[^a-zA-Z0-9](lib|include)[^a-zA-Z0-9]/) }}
```

Comment:
Checking for `lib` or `include` is not so common pattern to be here, and it will make this check useless as these 2 lines are checked, the conditions has AND between each line.